pbipa/1.8.0/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM mambaorg/micromamba:1.5.8 as app
+
+ARG IPA_VER="1.8.0"
+
+SHELL [ "/bin/bash", "-c" ]
+USER root
+
+WORKDIR /
+
+LABEL base.image="mambaorg/micromamba:1.5.8"
+LABEL dockerfile.version="1"
+LABEL software="IPA"
+LABEL software.version="${IPA_VER}"
+LABEL description="PacBio Improved Phased Assembler"
+LABEL website="https://github.com/PacificBiosciences/pbipa"
+LABEL license="https://github.com/PacificBiosciences/pbipa/blob/master/LICENSE.txt"
+LABEL maintainer="Kutluhan Incekara"
+LABEL maintainer.email="kutluhan.incekara@ct.gov"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    time \
+    procps && \    
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+RUN micromamba install --name base -c conda-forge -c bioconda python=3.8 pbipa=${IPA_VER} && \    
+    eval "$(micromamba shell hook --shell bash)" &&\
```

Comment:
This line indicates that this was painful to install. Thank you for your hard work!